lib/rpmsg/rpmsg_virtio.c

```diff
@@ -333,6 +333,8 @@ static void *rpmsg_virtio_get_tx_payload_buffer(struct rpmsg_device *rdev,
 		metal_mutex_release(&rdev->lock);
```

Comment:
Do we need to hold the lock, or re-lock the device during rpmsg_virtio_wait_tx_buffer()?

Reply:
No need, you can assume the simplest user's implementation of wait_tx_buffer() is metal_sleep_usec().